.devcontainer/post-start.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+echo "post-start started" >> $HOME/status
+
+# this runs each time Codespaces starts
```

Comment:
Should the patch scripts should be run in post-start?  Once the container shutdown and one comes back in the IPs no longer match.  So, I was thinking this might be a more automatic solution.  I'm also thinking the same thing about the jumpbox deployment for akdc, because I often have to delete and re-deploy it when restarting the Codespace due to IP changes.

Reply:
the IP shouldn't change between starts